static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,30 @@ define([
         }
     }
 
+    function paidforCardLoaded(el) {
+        var height,
+            $adSlot = $(el);
```

Comment:
At the moment we have two styles. Dotcom use the multi-var approach quite extensively, and we've adopted multi-var in files like `dfp-api` too. We should probably choose just one.

I know a few people aren't all that keen on the single-var format and I'd probably count myself amongst them. If nothing else, I'd say that single-var will become an antipattern in ES6, because it becomes less obvious in large declarations whether an entity is a `var`, a `let` or a `const`. I also think it's more awkward to write, and that having to swap semi-colons for commas when adding variables adds white noise to git diffs.

> Does it mean that I have to use that approach?

I don't think it matters that much for short declarations - it's when you have a screen full of vars that I think it gets awkward to work with. We did discuss how hard we should enforce consistencies like this at the last dotcom-hygiene meeting, but we didn't have a clear answer.